### ISSUE TITLE
iOS build failed

### ISSUE THREAD

**Issue by kleinpetr** [KasemJaffer/receive_sharing_intent#14]
I have got this error message via iOS building

```
  2 warnings generated.
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/receive_sharing_intent-1.3.0/ios/Classes/SwiftReceiveSharingIntentPlugin.swift:69:84: error: 'UIApplicationOpenURLOptionsKey' has been renamed to 'UIApplication.OpenURLOptionsKey'
        public func application(_ application: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
                                                                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                                       UIApplication.OpenURLOptionsKey
    UIKit.UIApplicationOpenURLOptionsKey:2:18: note: 'UIApplicationOpenURLOptionsKey' was obsoleted in Swift 4.2
    public typealias UIApplicationOpenURLOptionsKey = UIApplication.OpenURLOptionsKey
                     ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/receive_sharing_intent-1.3.0/ios/Classes/SwiftReceiveSharingIntentPlugin.swift:69:17: warning: instance method 'application(_:open:options:)' nearly matches optional requirement 'application(_:open:options:)' of protocol 'FlutterPlugin'
        public func application(_ application: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
                    ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/receive_sharing_intent-1.3.0/ios/Classes/SwiftReceiveSharingIntentPlugin.swift:69:17: note: move 'application(_:open:options:)' to an extension to silence this warning
        public func application(_ application: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
                    ^
    Flutter.FlutterPlugin:43:26: note: requirement 'application(_:open:options:)' declared here
        optional public func application(_ application: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool
                             ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/receive_sharing_intent-1.3.0/ios/Classes/SwiftReceiveSharingIntentPlugin.swift:55:36: error: 'UIApplicationLaunchOptionsKey' has been renamed to 'UIApplication.LaunchOptionsKey'
            if let url = launchOptions[UIApplicationLaunchOptionsKey.url] as? URL {
                                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                       UIApplication.LaunchOptionsKey
    UIKit.UIApplicationLaunchOptionsKey:2:18: note: 'UIApplicationLaunchOptionsKey' was obsoleted in Swift 4.2
    public typealias UIApplicationLaunchOptionsKey = UIApplication.LaunchOptionsKey
                     ^
    /Users/builder/programs/flutter/.pub-cache/hosted/pub.dartlang.org/receive_sharing_intent-1.3.0/ios/Classes/SwiftReceiveSharingIntentPlugin.swift:57:58: error: 'UIApplicationLaunchOptionsKey' has been renamed to 'UIApplication.LaunchOptionsKey'
            } else if let activityDictionary = launchOptions[UIApplicationLaunchOptionsKey.userActivityDictionary] as? [AnyHashable: Any] { //Universal link
                                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                             UIApplication.LaunchOptionsKey
    UIKit.UIApplicationLaunchOptionsKey:2:18: note: 'UIApplicationLaunchOptionsKey' was obsoleted in Swift 4.2
    public typealias UIApplicationLaunchOptionsKey = UIApplication.LaunchOptionsKey
```

**Comment by kleinpetr** [comment]
@KasemJaffer What do you mean? I have set Swift language version in Share Extension to 4.2..  Building with iosSDK 12.2. Xcode 10.2

**Comment by fgatti675** [comment]
I have the same issue. 
I am using SWIFT_VERSION 5
Seems just a matter of updating those obsolete references

**Comment by KasemJaffer** [comment]
Fixed in the latest version v1.3.3
